azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/persona/PersonaData.java

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.persona;
+
+import android.graphics.Bitmap;
+import android.widget.ImageView;
+
+import androidx.annotation.Nullable;
+
+import com.azure.android.communication.ui.configuration.LocalParticipantConfiguration;
+
+/**
+ * PersonaData for local participant configuration.
+ *
+ * <pre>
+ *
+ * &#47;&#47; Initialize the call composite builder
+ * final CallCompositeBuilder builder = new CallCompositeBuilder&#40;&#41;;
+ *
+ * &#47;&#47; Build the call composite
+ * CallComposite callComposite = builder.build&#40;&#41;;
+ *
+ * &#47;&#47; Build the LocalParticipantConfiguration with PersonaData
+ * LocalParticipantConfiguration config = new LocalParticipantConfiguration(new PersonaData&#40;...&#41);
+ *
+ * </pre>
+ *
+ * @see LocalParticipantConfiguration
+ */
+public class PersonaData {
+    private Bitmap image;
+    private String name;
+    private ImageView.ScaleType scaleType = ImageView.ScaleType.FIT_XY;
+
+    /**
+     * Create PersonaData.
+     *
+     * @param image The {@link Bitmap};
+     * @see LocalParticipantConfiguration
+     */
+    public PersonaData(final Bitmap image) {
```

Comment:
Parameter `image` is not clear. Should we call it `avatar` ?

Reply:
How about `avatarImage`?

Reply:
changed to  `avatarBitmap`